### PR TITLE
fix ci badge and add coverage

### PR DESCRIPTION
**Pull request by markcty**

### CODE REVIEW

**Comment by markcty** [comment]
WIP

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,83 +1,101 @@
...
+  schedule:
+    - cron: "00 19 * * *" # run ci periodically at 3 am
   pull_request:
     branches: [master]
 
 env:
   CI_RUST_TOOLCHAIN: 1.61.0
 
 jobs:
-  outdated:
```
Comment (rogercloud) [inline .github/workflows/ci.yml:11]: Why do you comment out all these jobs?

Comment (markcty) [reply .github/workflows/ci.yml]: Work In Progress. Had to open pr for the ci to run. Have reverted now

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,93 +1,95 @@
...
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/install@v0.1
-        with:
-          crate: cargo-outdated
-          version: latest
-      - run: cargo outdated
+   outdated:
+     name: Outdated
```
Comment (rogercloud) [inline .github/workflows/ci.yml:14]: remove the white space if it's not necessary. It's a meaningless change.

Comment (markcty) [reply .github/workflows/ci.yml]: Fixed.

**Review by rogercloud — APPROVED** [review]

**Pull request merged**